packages/backend/src/services/insight.service.ts

```diff
@@ -524,6 +526,14 @@ export class InsightService {
               }
             }
 
+            // Cleanup empty fields
+            if (mergedYaml.authors && mergedYaml.authors.length === 0) {
+              delete mergedYaml.authors;
+            }
+            if (mergedYaml.excludedAuthors && mergedYaml.excludedAuthors.length === 0) {
+              delete mergedYaml.excludedAuthors;
+            }
```

Comment:
This parts cleans up the YAML file and removes `authors: []` or `excludedAuthors: []`